### PR TITLE
Connectivity Tests: Bump test start delay

### PR DESCRIPTION
**Pull request by and-rewsmith**
Connectivity Tests encountered issues with not all modules being ready. We need to bump test start delay to mitigate. I am removing the plumbing as it adds overhead.

## Azure IoT Edge PR checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines and Best Practices
- [x] I have read the [contribution guidelines](https://github.com/azure/iotedge#contributing).
- [x] Title of the pull request is clear and informative.
- [x] Description of the pull request includes a concise summary of the enhancement or bug fix.

### Testing Guidelines
- [x] Pull request includes test coverage for the included changes.
- Description of the pull request includes 
	- [x] concise summary of tests added/modified
	- [x] local testing done.  

### Draft PRs
- Open the PR in `Draft` mode if it is:
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.

_Note: We use the kodiakhq bot to merge PRs once the necessary checks and approvals are in place. When it merges a PR, kodiakhq converts the PR title to the commit title, PR description to the commit description, and squashes all the commits in the PR to a single commit. The net effect is that entire PR becomes a single commit. Please follow the best practices mentioned [here](https://chris.beams.io/posts/git-commit/#:~:text=The%20seven%20rules%20of%20a%20great%20Git%20commit,what%20and%20why%20vs.%20how%20For%20example%3A%20) for the PR title and description_

### CODE REVIEW

**Comment by and-rewsmith** [comment]
Will wait on connectivity before merge:
https://dev.azure.com/msazure/One/_build/results?buildId=58028093&view=results

File: scripts/linux/trcE2ETest.sh
Comment (yophilav) [inline scripts/linux/trcE2ETest.sh:897]: So you want all the arch to have the delay of 6 minutes even AMD64?

Comment (and-rewsmith) [reply scripts/linux/trcE2ETest.sh:897]: The problem was observed on AMD64. And I think that is the only arch this script is called by. There is logic for arm, but afaik we don't use it.

**Comment by and-rewsmith** [comment]
New Run:
https://dev.azure.com/msazure/One/_build/results?buildId=58030691&view=results

**Review by yophilav — APPROVED** [review]

**Pull request merged**
